BilibiliLive/Component/Video/VideoPlayerViewController.swift

```diff
@@ -422,9 +422,12 @@ extension VideoPlayerViewController {
             }
         }
         if let defaultRate = self.player?.defaultRate,
-           let speed = PlaySpeed.blDefaults.first(where: { $0.value == defaultRate }) {
+           let speed = PlaySpeed.blDefaults.first(where: { $0.value == defaultRate })
+        {
             self.player = player
             selectSpeed(AVPlaybackSpeed(rate: speed.value, localizedName: speed.name))
+        } else {
+            self.player = player
```

Comment:
After I merged the latest main I found I could not play any video. Checking all the new commits, I found it was brought by c71bc19a418aef7a79622b37fe3563733afc5106

I deleted the 2 lines by mistake when bumping the tvOS 16 target.